funnel/views/commentvote.py

```diff
@@ -173,6 +174,64 @@ def new_comment(self):
         )
         return {'form': commentform_html}
 
+    @route('subscribe', methods=['POST'])
+    @requires_login
+    @render_with(json=True)
+    def subscribe(self):
+        csrf_form = forms.Form()
+        if csrf_form.validate_on_submit():
+            if self.obj not in current_auth.user.subscribed_commentsets:
+                new_ms = CommentsetMembership(
+                    commentset=self.obj,
+                    user=current_auth.user,
+                    is_subscriber=True,
+                    granted_by=current_auth.user,
+                )
+                db.session.add(new_ms)
+                db.session.commit()
+            return {
+                'status': 'ok',
+                'message': _("You have successfully subscribed to this comment thread"),
```

Comment:
We are no longer using first and second person pronouns to refer to the user. Text has to change.